packages/labs/cli/src/bin/lit.ts

```diff
@@ -0,0 +1,29 @@
+#!node
```

Comment:
The shebang should be `#!/usr/bin/env node` right? Because PATH is not consulted when evaluating shebangs.

Reply:
I think you're right, but this works for me. Changed.